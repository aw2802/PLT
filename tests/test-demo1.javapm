public class TestWhileForNested {
  public void main(){
    int i = 0;
    int j;
    while(i < 2){
      for(j = 0; j < 2; j = j + 1){
      	println("In a for in a while!");
      }  
      i = i + 1;
    }
    println();
    
    for(i = 0; i < 2; i = i + 1){
      j = 0;
      while(j < 2){
        println("In a while in a for!");
	j = j + 1;
      }
    }
  } 
}
