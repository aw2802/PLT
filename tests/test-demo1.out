In a for in a while!
In a for in a while!
In a for in a while!
In a for in a while!


In a while in a for!
In a while in a for!
In a while in a for!
In a while in a for!
